src/connector/src/sink/log_store.rs

```diff
@@ -204,9 +204,11 @@ impl<R: LogReader> LogReader for MonitoredLogReader<R> {
                 self.metrics.log_store_latest_read_epoch.set(*epoch as _);
             }
             if let LogStoreReadItem::StreamChunk { chunk, .. } = item {
+                let cardinality = chunk.cardinality();
+                self.metrics.log_store_read_rows.inc_by(cardinality as _);
                 self.metrics
```

Comment:
The metrics was originally reported only in connector node. So it's better to report it only at the remote sink.

Reply:
Let me open another PR to add this metric.